src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const sourceStringTrim = sourceString.trim();
+  const tabFromSourceString = sourceStringTrim.split(';');
+
+  for (let i = 0; i < tabFromSourceString.length; i++) {
+    const tabProperty = tabFromSourceString[i].split(':');
+
+    if (tabProperty.length === 2) {
+      const key = tabProperty[0].trim();
+      const value = tabProperty[1].trim();
+
+      if (key && value) {
+        obj[key] = value;
+      }
+    }
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Consider using `split` to create an array of style declarations and then `reduce` to transform this array into an object.